.travis.yml

```diff
@@ -22,6 +22,8 @@ script:
 
 deploy:
   provider: script
-  script: ./scripts/push.sh
+  script: ./scripts/push.sh production $TRAVIS_TAG
+  skip_cleanup: true
   on:
-    tags: false
+    branch: master
+    tags: true
```

Comment:
With `branch: master` and `tags: true` both set, will there be a chance that a release is pushed out twice, once for the current `master` and once for the new tag?